server/app.js

```diff
@@ -29,7 +29,7 @@ const io = new Server(server, {
   },
 });
 
-require('./utils/socketServer')(io);
+// require('./utils/socketServer')(io);
```

Comment:
Any particular reason to comment this out?

Reply:
I left it commented out from before, I undid it now